components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -84,6 +85,12 @@ struct SiteInfo {
   // Web page specific fields, if available and allowed
   string? title;
   string? hostname;
+
+  // Note: iOS doesn't support optional int32s in mojom, so we use -1 to
+  // indicate that the content id is not set.
+  // This is the ID of the content currently loaded in the tab.
+  int32 content_id;
```

Comment:
should SiteInfo maybe have a TabData member instead of duplicating this?

Reply:
maybe it doesn't make sense here, just want to check

Reply:
https://github.com/brave/brave-core/pull/27243

I've got a PR to clean this up doing more or less just this :smile: